src/domain/categories.ts

```diff
@@ -5,21 +5,18 @@ export type Category = {
   title: string;
 };
 
+export type CategoryRequest = Omit<Category, "id">;
+
 export function fetchData() {
-  return fetch(`http://localhost:3000/category`)
-    .then((result) => {
-      return result.json();
-    })
-    .then((data: Category[]) => {
-      return [...data];
-    });
+  return fetch(`http://localhost:3000/category`).then((result) => {
+    return result.json();
+  });
```

Comment:
Disini kamu bisa kasih typing `Category[]` biar di component gak perlu nambahin secara manual 

```suggestion
  return fetch(`http://localhost:3000/category`)
    .then((result) => result.json())
    .then((data: Category[]) => data);
```

Reply:
done update